crates/api_models/src/admin.rs

```diff
@@ -1973,7 +1973,7 @@ pub struct ProfileCreate {
 
     /// Product authentication ids
     #[schema(value_type = Option<Object>, example = r#"{ "key1": "value-1", "key2": "value-2" }"#)]
-    pub authentication_product_ids: Option<HashMap<String, id_type::MerchantConnectorAccountId>>,
+    pub authentication_product_ids: Option<common_types::payments::MerchantConnectorAccountMap>,
```

Comment:
Can we please rename this something specific to Authentication, instead of a generic map?

Reply:
but this type can be used for other purposes as well! would it be wise to create multiple redundant type serving same purpose?

Reply:
In the Hashmap the key is `String` and if we make a newtype called `Product` means then the Map is specific to this use case, and we don't be creating redundant type.

So we shouldn't use  generic term like `MerchantConnectorAccountMap`.